modules/installation/pages/client-requirements.adoc

```diff
@@ -6,20 +6,27 @@
 
 Supported operating systems for traditional and Salt clients are listed in this table.
 
+In this table, &#2713; indicates that clients running the operating system are supported by {scc}, and &#2717; indicates that it is not supported.
```

Comment:
Please check if {scc} is correct here. I do not now what the replacement is, but "supported by SUSE Customer Center" does not sound right.

A second thing to check are the unicode characters, Here in github they show something different than wanted.

Reply:
Updated to "supported by {suse}".

Reply:
Thanks